README.md

```diff
@@ -91,7 +103,10 @@ Write this grammar using EBNF with common extensions
 
 ## A2.2
 
-«replace this with your answer»
+< adjective >     ::=     [{"lazy"|"smelly"}]  
+< noun >          ::=     "dog"|"cat"  
+< verb >          ::=     "ate"|"ran"  
+< adverb >        ::=     ["slowly"|"noisily"]  
```

Comment:
# 3

Again no sentence. And what is `[{"lazy"|"smelly"}]`? {} means zero or more, and [] means optional.